URBNSwiftAlert/Classes/URBNSwAlertStyler.swift

```diff
@@ -86,7 +86,13 @@ public struct URBNSwAlertStyler {
     public var animationInitialVelocity: CGFloat = -10.0
     
     /**
-     * Margins on left and right sides of the alert, which sets the width constraint.  Used if no insets are given.  The alert view will use its most compressed layout values for height
+     * Min (greaterThanOrEqualTo) and Max widths (lessThanOrEqualTo) for the Alert -   The alert view will use its most compressed layout values for height
+     */
+    
+    public var minMaxWidth: (CGFloat, CGFloat)?
```

Comment:
i dig the tuple. Not that this should be a deciding factor, but that will not bridge to objc correct? You can also name the params in the tuple which may help, i.e. `(min: CGFloat, max: CGFloat)` then when you goto use then, you can actually do `.min` for example rather than `.0`